### PR TITLE
#48733 added logging statements to LLVM codegen using JIT logging

### PR DESCRIPTION
**Pull request by huiguoo**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#48758 #48733 added logging statements to LLVM codegen using JIT logging**

Differential Revision: [D25295995](https://our.internmc.facebook.com/intern/diff/D25295995)

### CODE REVIEW

**Review by ZolotukhinM — APPROVED** [review]
Awesome, thanks for adding this!

**Comment by ZolotukhinM** [comment]
In case you'd like to improve the LLVM codegen logging even further, I've just filed one more simple issue: #48771. Please feel free to grab it if you are interested!

**Comment by huiguoo** [comment]
Thanks @ZolotukhinM ! Will be happy to further improve the LLVM codegen logging.

File: torch/csrc/jit/tensorexpr/llvm_codegen.cpp [hunk 1]
```diff
@@ -518,6 +519,13 @@ void LLVMCodeGenImpl::emitKernel(
   if (llvm::verifyFunction(*fn_, &llvm::outs())) {
     throw std::runtime_error("Function verification failed");
   }
+
+  // print graph debug info.
+  std::string fnstr;
```
Comment (eellison) [inline torch/csrc/jit/tensorexpr/llvm_codegen.cpp:524]: It's not ideal that we're dumping the llvm into a string even if graph_debug isn't on, slowing down all compilation

Comment (huiguoo) [reply torch/csrc/jit/tensorexpr/llvm_codegen.cpp:524]: Agree. This was fixed in a follow-up issue (issue #48771, commit e99d7df).

**Pull request closed without merging**